src/EditorFeatures/CSharpTest/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -29,33 +29,36 @@ public async Task TestEmptyFile()
             await VerifyCS.VerifyRefactoringAsync(code, code);
         }
 
-        [Theory, Trait(Traits.Feature, Traits.Features.CodeActionsInitializeParameter)]
-        [InlineData("", true)]
-        [InlineData("csharp_style_prefer_parameter_null_checking = true", true)]
-        [InlineData("csharp_style_prefer_parameter_null_checking = false", false)]
-        public async Task TestSimpleReferenceType(string editorConfig, bool useParameterNullChecking)
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInitializeParameter)]
+        public async Task TestSimpleReferenceType()
```

Comment:
It looks like the case where the option is explicitly enabled was removed. Was that intended?